cmd/minishift/cmd/start.go

```diff
@@ -188,6 +190,8 @@ func runStart(cmd *cobra.Command, args []string) {
 		preflightChecksBeforeStartingHost()
 	}
 
+	populateStartFlagsToViperConfig()
```

Comment:
@praveenkumar IMO the name of the function should be `populateStartFlagsToConfig()`.